app/src/components/integrations/integrationProviders/rallyIntegration/rallyConnectionFormFields/rallyConnectionFormFields.jsx

```diff
@@ -81,35 +81,31 @@ export class RallyConnectionFormFields extends Component {
           disabled={disabled}
           label={formatMessage(COMMON_BTS_MESSAGES.integrationNameLabel)}
           className={cx('fields')}
+          isRequired
         >
           <FieldErrorHint provideHint={false}>
-            <FieldText
-              placeholder={formatMessage(COMMON_BTS_MESSAGES.integrationNameLabel)}
-              defaultWidth={false}
-              maxLength={55}
-              isRequired
-            />
+            <FieldText defaultWidth={false} maxLength={55} />
           </FieldErrorHint>
         </FieldElement>
         <FieldElement
           name="url"
           validate={commonValidators.btsUrl}
           disabled={disabled || editAuthMode}
           label={formatMessage(COMMON_BTS_MESSAGES.linkToBtsLabel)}
-          placeholder={formatMessage(COMMON_BTS_MESSAGES.linkToBtsLabel)}
           className={cx('fields')}
+          isRequired
         >
           <FieldErrorHint provideHint={false}>
-            <FieldText defaultWidth={false} isRequired />
+            <FieldText defaultWidth={false} />
           </FieldErrorHint>
         </FieldElement>
         <FieldElement
           name="project"
           validate={commonValidators.btsProjectId}
           disabled={disabled || editAuthMode}
           className={cx('fields')}
-          placeholder={formatMessage(messages.projectIdLabel)}
           label={formatMessage(messages.projectIdLabel)}
+          isRequired
         >
           <FieldErrorHint provideHint={false}>
             <FieldText defaultWidth={false} maxLength={55} isRequired />
```

Comment:
Should it be removed here - ` isRequired` ?

Reply:
No, thanks. It doesn't affect anything so I didn't see.